test/regression/aarch64/AArch64RegressionTest.hh

```diff
@@ -6,6 +6,7 @@
 
 #define AARCH64_CONFIG                                                         \
   ("{Core: {Simulation-Mode: emulation, Clock-Frequency: 2.5, "                \
+   "Timer-Frequency: 100, "                                                    \
```

Comment:
Could we have this formatted correctly (i.e. have ends of multi-line string more-or-less aligned)

Reply:
I tried my best to do this, but clang format would not recognise it as a `define` and produced incorrect code. I will try again to format it correctly